docs/resources/cdn_domain.md

```diff
@@ -373,14 +383,52 @@ The `url_signing` block support:
   **type_c1**: Method C1.
   **type_c2**: Method C2.
 
-* `key` - (Optional, String) Specifies the authentication key contains `6` to `32` characters, including letters and digits.
+* `sign_method` - (Optional, String) Specifies the encryption algorithm type for URL authentication.
+  The valid values are as following:
+  + **md5**
+  + **sha256**
+
+* `match_type` - (Optional, String) Specifies the authentication scope.
+  Currently, only spuuort value is **all**, indicates that all files are involved in authentication.
```

Comment:
The word is `support`, and there's something wrong with its syntax.

Reply:
All optional values ​​need to add default value description, please check all documents.